tutorials/asr/09_ASR_for_telephony_speech.ipynb

```diff
@@ -0,0 +1,423 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "audio_8kHz = librosa.core.resample(audio, sample_rate, 8000)\n",
+    "spec = np.abs(librosa.stft(audio_8kHz))\n",
+    "spec_db = librosa.amplitude_to_db(spec, ref=np.max)  # Decibels\n",
+    "\n",
+    "# Use log scale to view frequencies\n",
+    "librosa.display.specshow(spec_db, y_axis='log', x_axis='time',  sr=8000)\n",
+    "plt.colorbar()\n",
+    "plt.title('Audio Spectrogram');\n",
+    "plt.ylim([0, 8000])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import soundfile as sf\n",
+    "sf.write(data_dir + '/audio_16kHz.wav', audio_16kHz, 16000)\n",
+    "sf.write(data_dir + '/audio_8kHz.wav', audio_8kHz, 8000)"
```

Comment:
Maybe use IPython.display.Audio() to allow user to listen to these two clips for audible differences?